Common/Data/Custom/PsychSignal/PsychSignalSentiment.cs

```diff
@@ -29,6 +29,26 @@ namespace QuantConnect.Data.Custom.PsychSignal
     /// </summary>
     public class PsychSignalSentiment : BaseData
     {
+        /// <summary>
+        /// The EndTime of the bar represents the time the data ended and
+        /// the time it should be emitted in backtesting/live trading
+        /// </summary>
+        public override DateTime EndTime {
+            get { return Time.Add(Period).Add(FixedLiveOffset); }
+            set { Time = value.Subtract(Period); }
```

Comment:
This property syntax doesn't seem to follow the rest of the code base which would place the opening brace on it's own line. What you have here is reminiscent of the K&R bracing style common in c/c++ and javascript